odata-url-conventions/5 Query Options.md

```diff
@@ -1956,9 +1946,6 @@ rules, in order:
 Each of these promotions uses the same semantics as a `castExpression`
 to promote an operand to the target type.
 
-OData does not define an implicit conversion between string and numeric
```

Comment:
Deleted that, because it contradicts the statement in the `cast` function section:
* It flatly contradicted the old statement (`cast` is optional).
* It still contradicts the now-restricted statement (implicit conversion only for parameters).